lib/ssl/src/ssl_crl_cache.erl

```diff
@@ -72,9 +72,20 @@ fresh_crl(#'DistributionPoint'{distributionPoint = {fullName, Names}}, CRL) ->
 %% API 
 %%====================================================================
 
+%%--------------------------------------------------------------------
+-spec insert(CRLSrc) -> ok | {error, Reason} when
+      CRLSrc :: crl_src(),
+      Reason :: term().
+%%--------------------------------------------------------------------
 insert(CRLs) ->
     insert(?NO_DIST_POINT, CRLs).
```

Comment:
If not a big deal, maybe align naming have either CRLSrc or CRLs depending on which is better.

Reply:
Made a change